### PR TITLE
[README] Add README

### PR DESCRIPTION
**Pull request by dennis1248**
Closes https://github.com/manjaro/arkdep-variants/issues/9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Manjaro Arkdep variants
```
Comment (romangg) [inline README.md:1]: Title case

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+- Have arkdep-build and dependencies installed.
+
+```bash
+sudo pacman -Sy arkdep
+```
+
+## Building a variant
+
+First clone the repo and navigate to its root;
```
Comment (romangg) [inline README.md:17]: . or :

Comment (romangg) [reply README.md:17]: I see you later used semicolon again. Not an accident?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+cd arkdep-variants
+```
+
+Variants are identified based on directory name. Pass the directory name of whichever variant you wish to build to arkdep-build to build it;
+
+```bash
+sudo arkdep-build test-manjaro-gnome
+```
+
+Once the image finished building they will be deposited in `./target`.
```
Comment (romangg) [inline README.md:30]: Comma?

**Review by romangg — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (dennis1248) [reply README.md:17]: That is my personal writing style. Means "Look at thing below". I can change that.

File: README.md (hunk 3 above)
Comment (dennis1248) [reply README.md:30]: Well spotted!

**Pull request merged**